### DISCUSSION TITLE
Preset naming

### DISCUSSION THREAD

**Discussion by shanekuz** [blakeblackshear/frigate#11440] [Ideas]
Is there anyway in the config that I can put in the onvif config names for each preset so that the drop down in the UI says names like "front gate" etc?  

onvif:
  presets_enable: true
     1: Front Gate
     2: Back Gate

etc. you get the idea

Thanks for a great product.

**Comment by NickM-27** [comment]
Presets are set on the camera, so name them whatever you want on the camera itself

**Reply by shanekuz** [reply]
I have 5 different cameras and none allow naming in the camera and they all just present as 1 - 10 in the UI.  Is this a document ONVIF standard or just works with some cameras?

**Reply by NickM-27** [reply]
Seems like a camera implementation issue. My amcrest camera allows defining names and I know reolink cameras do as well

**Reply by izzygold** [reply]
A lot of lesser quality cameras do not support this, I have the same issue. any chance to reconsider implementing this?

**Comment by peteDDD** [comment]
For what it is worth... I am converting from Blue Iris to Frigate.   My cameras which do not allow assignment of names to presets (SV3C cameras) had names to the presets applied in Blue Iris which carried through to the android app for Blue Iris.  It would seem since it can be done, that having an override of the presets' names available in the frigate.yml would be a useful feature.
